include/deal.II/base/tensor_base.h

```diff
@@ -1,1552 +0,0 @@
-// ---------------------------------------------------------------------
```

Comment:
Do I read this patch correctly that tensor_base.h is completely removed? Since tensor_base.h is user-visible, don't we need to keep a dummy (with deprecation warning) for at least one release or so?